### ISSUE TITLE
[Feature request] Add horizontal scroll support for document tabs

### ISSUE THREAD

**Issue by ioparin06** [notepad-plus-plus/notepad-plus-plus#14233]
While hovering over the opened tabs, using mouse horizontal scroll (like Mx Master 3 mouse) does not make anything. Would be great to apply the browsers tabs like behaviour, when horizontal scroll moves tabs left and right (not changing the selected tab, just scrolling).

Debug info:
```
Notepad++ v8.5.7   (64-bit)
Build time : Sep  6 2023 - 23:13:49
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 Pro (64-bit)
OS Version : 22H2
OS Build : 22621.2283
Current ANSI codepage : 1252
Plugins : 
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4) 
```

**Comment by alankilborn** [comment]
Which browser are you talking about?
I tried pointing at existing tabs in Chrome and Edge, and then scrolling my Mx Master 3's horizontal scroll wheel, and exactly nothing happened.

**Comment by ioparin06** [comment]
@alankilborn You're right, I've tried that in Chrome and it does not do anything. But with Firefox it does. If you add so many tabs, so that they do not fit in the available width, horizontal scroll makes sense, can you try that please?

**Comment by alankilborn** [comment]
In Notepad++, if you point the mouse at the tabs and then hold Ctrl while scrolling the main mouse wheel, it changes the tab in the indicated direction.  Note that this is a bit different than your request, as you said "not changing the selected tab".

**Comment by alankilborn** [comment]
>The behaviour you describe is possible to get by customizing Mx Master 3 mouse through their Logi Options app.

No, I have not installed any "Logi Options" app.  What I described is behavior that will work for any mouse -- with a (vertical) wheel.

>instead of the requesting feature I'll say: is it possible (and does it make sense) to make scroll availavle through the whole tabs line and not only when the mouse is over one of them?

It probably does not make sense, and here's why:  If there is "blank space" on the right in the tab bar, it very likely means that all of the tabs you have open are already visible -- so just click on the one you want to make active.

**Comment by ioparin06** [comment]
> it very likely means that all of the tabs you have open are already visible
Not really. In the screenshot I sent above on the left side of the most-left visible tab "new10" there is also a ton of opened tabs

**Comment by alankilborn** [comment]
>Not really. In the screenshot I sent above on the left side of the most-left visible tab "new10" there is also a ton of opened tabs

Ah, okay; yea, you're right.  Sorry.
I'm just not familiar with this mode, as I've used Notepad++ for 10 years with "multiline" tabs turned on.

Still, I think is likely that the Notepad++ developers won't go for making a change such that pointing at the smallish "tab white space" and then scrolling with a mouse wheel will do this.  It's just simple enough to point AT the tabs and do it.
